cypress/support/pages/setting.pageObject.js

```diff
@@ -0,0 +1,57 @@
+import PageObject from '../PageObject';
+
+class SettingPageObject extends PageObject {
+  get userNameField() {
+    return cy.getByDataCy('Username').clear();
+  }
+
+  get emailField() {
+    return cy.getByDataCy('Email').clear();
+  }
+
+  get passwordField() {
+    return cy.getByDataCy('New Password').clear();
+  }
+
+  get updateButton() {
+    return cy.get('form > :nth-child(1) > .btn');
+  }
+
+  get logoutButton() {
+    return cy.get('.btn-outline-danger');
```

Comment:
The CSS selector for `logoutButton` is specific and could be fragile if the HTML structure changes. Consider using a more robust selector, such as a data attribute, if available.